File: swagger/models/client_relationship.py
from .base_model import BaseModel

from .relationship import Relationship

class ClientRelationship(BaseModel):
	"""
	A relation between two clients.

	Attributes:

		related_client_id
		The RSSID of the related client.

		relationship
		Contains details about the relationship between two clients.

		relationship_name
		The name of the relationship of the related client.

		delete
		When true, this relationship is removed from the associated clients.
	"""

	input_map = {
		'RelatedClientId': 'related_client_id',
		'Relationship': ('relationship', Relationship)
		'RelationshipName': 'relationship_name',
		'Delete': 'delete',
		}

	output_map = {
		'related_client_id': 'RelatedClientId',
		'relationship': 'Relationship',
		'relationship_name': 'RelationshipName',
		'delete': 'Delete',
		}

File: swagger/Models/ClientRelationship.php
<?php

namespace App\MindBody\Models;

/**
 * A relation between two clients.
 */
class ClientRelationship extends BaseModel
{
	/**
	 * The RSSID of the related client.
	 * 
	 * @var string
	 */
	public $relatedClientId;
	/**
	 * Contains details about the relationship between two clients.
	 * 
	 * @var Relationship
	 */
	public $relationship;
	/**
	 * The name of the relationship of the related client.
	 * 
	 * @var string
	 */
	public $relationshipName;
	/**
	 * When true, this relationship is removed from the associated clients.
	 * 
	 * @var boolean
	 */
	public $delete;

	protected function getInputMap()
	{
		return [
			'RelatedClientId' => 'relatedClientId',
			'Relationship' => ['relationship', Relationship::class],
			'RelationshipName' => 'relationshipName',
			'Delete' => 'delete',
			];
	}

	protected function getOutputMap()
	{
		return [
			'relatedClientId' => 'RelatedClientId',
			'relationship' => 'Relationship',
			'relationshipName' => 'RelationshipName',
			'delete' => 'Delete',
			];
	}
}
